sycl/test/gdb/accessors-device.cpp

```diff
@@ -1,19 +1,51 @@
-// RUN: %clangxx -fsycl-device-only -c -fno-color-diagnostics -Xclang -ast-dump %s -I %sycl_include -Wno-sycl-strict | FileCheck %s
+// RUN: %clangxx -fsycl-device-only -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s -I %sycl_include -Wno-sycl-strict | FileCheck %s
 // UNSUPPORTED: windows
-#include <sycl/accessor.hpp>
+#include <sycl/sycl.hpp>
 
 typedef sycl::accessor<int, 1, sycl::access::mode::read> dummy;
```

Comment:
Is this being used anywhere?

Reply:
I think you're right, let me try to remove it.